src/components/App.js

```diff
@@ -17,22 +17,52 @@ import NotFound from './NotFound/NotFound'
 // router
 import { Route, Switch } from 'react-router'
 import { HashRouter } from 'react-router-dom'
+import { BarLoader } from 'react-spinners'
 
 class App extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      loading: true
+    }
+  }
+  componentDidMount () {
+    // alert('mounted')
+    this.setState({
+      loading: false
+    })
+  }
   render () {
+    const renderApp = () => {
+      if (this.state.loading) {
+        return (
+          <div className='sweet-loading'>
+            <BarLoader
+              color='#297FCA'
+              loading={this.state.loading}
```

Comment:
You made `if (this.state.loading) {` and the you made `loading={this.state.loading}` - this doesn't have a sense